### ISSUE TITLE
Вопрос о 2 тестировании в МГТУ

### ISSUE THREAD

**Issue by Avonna** [Velervv/VKR#1]
Здравствуйте.  
Прошу прощения за беспокойство. Я сейчас обучаюсь на курсе Data Science в МГТУ и прохожу второе тестирование. Там есть задание с датасетом: #Разработать и обучить нейронную сеть на TensorFlow для решения задачи классификации рукописных цифр. Набор данных: MNIST, источник: from keras.datasets import mnist 
 
#В нейронной сети должно быть 3 Dense-слоя: два скрытых слоя, состоящих из 128 нейронов каждый, и выходной слой. Активационные функции на скрытых слоях relu, на выходном - softmax. 
 
#В качестве ответа необходимо указать количество оптимизируемых параметров (Trainable params) в виде целого числа 
#и прикрепить к ответу подробное решение с кодом в виде файла или архива с файлами.

Хотела спросить, вдруг у вас осталось решение. Правильный ли ответ: Total params: 147,402
Trainable params: 147,402
Non-trainable params: 0
__________________________Благодарю.

**Comment by Velervv** [comment]
Добрый день, коллега!)
Сейчас посмотррю)

**Comment by Velervv** [comment]
https://drive.google.com/file/d/1YII0nrwaUFWfb__aDHs1HkVgteF0oLCJ/view?usp=sharing
здесь код.
ответ у нас получался 118282

**Comment by Avonna** [comment]
Спасибо большое! Поняла, где ошибку сделала. Исправила, все заработало. Пока тест еще на оценивании, но хочется верить, что все будет хорошо. Много раз благодарю Вас за помощь. Удачи Вам и всего самого хорошего

**Comment by Velervv** [comment]
Всегда рада помочь) Верю, что у вас все пройдет хорошо!

**Comment by Avonna** [comment]
Спасибо большое и Вам всего самого-самого наилучшего.
